### ISSUE TITLE
gx_oc_2 LV2 plugin is not available in guitarix GUI

### ISSUE THREAD

**Issue by ycollet** [brummer10/guitarix#90]
I looked for an octaver and first tried Detune.
I found after that a web page talkling about OC2.
The last version doesn't display the OC2 octaver plugin. But the source are in git repository of guitarix.
Can it be possible to add it to guitarix ? (For now, I add this plugin via the add LV2 menu).

**Comment by brummer10** [comment]
Yes, that is true. But that is true for any other LV2 plugin as well. The Info log on guitarix will inform you about that so it will be easy to load the missing plug. On the other hand, implement it as guitarix native plug will double the work in the long run maintenance of the source code. Hence I prefer to not implement it natively.

**Comment by ycollet** [comment]
The only problem I've got is that I discovered this plugin "by accident". 
I was ansolutely not think about looking into lv2-gxuitarix-plugins on Fedora because, for me, in that package I only found plugins corresponding to effect found  in the GUI.
I feel this plugin should be moved to GxPlugins.
Maybe some other LV2 plugiins only found in the source directory of guitarix (but I have not identified yet such plugins excepted oc_2) should be moved in Gxplugins.
